app/scripts/modules/core/src/pipeline/config/stages/rollbackCluster/RollbackClusterExecutionDetails.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+
+import { ExecutionDetailsSection, IExecutionDetailsSectionProps } from 'core/pipeline/config/stages/core';
+import { HelpField } from 'core/help/HelpField';
+import { NgReact } from 'core/reactShims';
+import { StageExecutionLogs, StageFailureMessage } from 'core/pipeline/details';
+
+export function RollbackClusterExecutionDetails(props: IExecutionDetailsSectionProps) {
+  const { AccountTag } = NgReact;
+  const { stage } = props;
+
+  const imagesToRestore = stage.context.imagesToRestore && stage.context.imagesToRestore.map((entry: any) => {
+    const imageDetails = entry.buildNumber ? '#' + entry.buildNumber : entry.image;
+    const method = (entry.rollbackMethod === 'EXPLICIT') ? 'previous server group' : 'new server group';
+    const helpId = 'cluster.rollback.' + entry.rollbackMethod.toLowerCase();
+    return (
+      <div key={entry.region}>
```

Comment:
Had a lint error without the `key` ... not sure if I'm doing something wrong.

Reply:
The `key` is to identify the uniqueness of the element in the array created by `stage.context.imagesToRestore.map((entry: any) => {}`. As long as the key you choose is guaranteed to be unique across that entire map, then you're doing something right!